malariagen_data/anoph/snp_data.py

```diff
@@ -1565,8 +1565,12 @@ def biallelic_snp_calls(
         min_cohort_size: Optional[base_params.min_cohort_size] = None,
         max_cohort_size: Optional[base_params.max_cohort_size] = None,
         random_seed: base_params.random_seed = 42,
-        min_minor_ac: Optional[base_params.min_minor_ac] = None,
-        max_missing_an: Optional[base_params.max_missing_an] = None,
+        min_minor_ac: Optional[
+            base_params.min_minor_ac
+        ] = base_params.min_minor_ac_default,
+        max_missing_an: Optional[
+            base_params.max_missing_an
+        ] = base_params.max_missing_an_default,
```

Comment:
Suggest to leave defaults of None for this function, biallelic_snp_calls() is a more general function and so these defaults don't necessarily make sense.

Reply:
Will do. Thanks @alimanfoo .